CafeMenuEditor/mainwindow.cpp

```diff
@@ -110,6 +112,18 @@ void MainWindow::example()
     refreshMenu();
 }
 
+void MainWindow::aboutMenu()
+{
+    QMessageBox msg;
+    msg.information(this, "Menu Editor", "v.1.0", QMessageBox::Ok);
```

Comment:
QMessageBox::information() is a **static** method. So, you can use it without creating a new object. See [http://doc.qt.io/qt-5/qmessagebox.html#the-static-functions-api](http://doc.qt.io/qt-5/qmessagebox.html#the-static-functions-api) for more info.